### DISCUSSION TITLE
Contact for mastodon.social account recovery

### DISCUSSION THREAD

**Discussion by catleeball** [mastodon/mastodon#17142] [Q&A] [answered]
Hi everyone,

Does anyone know who I can reach out to at mastodon.social about recovering or deleting my old account from 2017? I lost the 2nd factor, and the recovery page says to email staff@mastodon.social.

![ad9ea00fec074ff09d3b167def9223e42e407c24](https://user-images.githubusercontent.com/43632885/146519454-98ac2c8d-2b52-4b08-900f-a7c2dc76cd86.png)

I did email staff@ on Nov 8th and 15th, but haven't heard back. Likewise, I emailed @Gargron at eugen@zeonfederated.com on Dec 8th. I also posted on the [old discourse](https://discourse.joinmastodon.org/t/mastodon-social-account-recovery-question/3984).

Is there anyone who I can contact regarding this? I still have access to my original email, and even have the original account confirmation emails sent from Mastodon when I signed up.

Thanks!

**Comment by Gargron** [comment, marked as answer]
Replied

**Reply by catleeball** [reply]
Thank you! :)

**Comment by niam0t** [comment]
I got a issue with my mastodon.social account. I wanted to change the related Mail from Gmail to ProtonMail, but in such process i lost my account somehow. Now it shows my previous username is taken and i can't login using previous or new Mail. Says Mail orPassword is wrong, tried recovering the account using both Mail, but I never get the Recovery Mail on any of the Mail.
How can i contact with mastodon for recovery process?

**Reply by renchap** [reply]
You can send an email to staff@mastodon.social
